frontend/src/app/components/global-search/global-search-title.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="toolbar-container">
+  <div class="toolbar">
+    <div class="title-container">
+      <h2 [attr.title]="searchTerm">
+        Search for
+        <i>"{{ searchTerm }}"</i>
+        {{ project !== '' ? 'in' : ''}}
+        <i>{{ project }}</i>
+        {{ projectScope === '' ? 'and all Subprojects' : ''}} </h2>
```

Comment:
Please use I18n. Please look at GlobalSearchInputComponent if you need an example.